gtfs/gtfs_utils/gtfs_utils/s3.py

```diff
@@ -57,8 +60,9 @@ def get_bucket_file_keys_for_date(crud: S3Crud,
 
 def get_latest_file(crud: S3Crud,
                     mot_file_name: str,
-                    desired_date: datetime.datetime,
-                    past_days_to_try: int = 100) -> Tuple[datetime.date, str]:
+                    desired_date: Union[datetime.datetime, datetime.date],
```

Comment:
Because we pass datetime.date from gtfs_stats, and I didn't like the PyCham warning :P

(`def get_dates_without_output(dates: List[datetime.date], output_folder: str) -> List[datetime.date]:`)

Reply:
Do we have a reason no to change it to simply `datetime.date instead`?